.tekton/copy-subscription-manager-activation-key-oci-ta.yaml

```
@@ -0,0 +1,55 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: copy-subscription-manager-activation-key-oci-ta
+  namespace: rh-acs-tenant
+# TODO(ROX-23812): Refactor to a task bundle
+spec:
+  description: Smuggles an activation key next to the application source code that enables access to RHEL RPMs.
+  params:
+  - name: SOURCE_ARTIFACT
+    description: The Trusted Artifact URI pointing to the application source code with
+      the additional smuggled activation key. This should be the result of the git-clone task,
+      results from other tasks might fail as dirty.
```

I realized that `params` and `results` `SOURCE_ARTIFACT` descriptions should be mostly swapped.
```suggestion
  params:
  - name: SOURCE_ARTIFACT
    description: The Trusted Artifact URI pointing to the application source code.
```